src/operator/subgraph/build_subgraph.cc

```diff
@@ -563,6 +563,18 @@ void CutGraphInputs(const std::vector<nnvm::NodeEntry*> &input_entries,
   }
 }
 
+/*!
+ * \brief This function reattaches the original input nodes that were cut
```

Comment:
is this function used for rejecting subgraph? you can add a short comment on the caller

Reply:
done, thanks!